tests/examplefiles/flex/example.l

```diff
@@ -0,0 +1,79 @@
+/* scanner for a toy Pascal-like language */
```

Comment:
For all these tests, you also need to add test output. This is how we catch regressions: the testing infrastructure compares the output of Pygments against the expected output. To generate this, run `pytest --update-goldens tests/examplefiles/flex` and commit the resulting `.output` files.